_posts/2020-03-29-newpipe-0.19.0-released.md

```diff
@@ -0,0 +1,67 @@
+
+## Fixes a.k.a. NewPipe crashes less \o/
+
+Andriod 9 changed how databases behave, effectively destroying backups and exporting remote playlists - not anymore, thanks to [@yausername](https://github.com/yausername). [@XiangRongLin](https://github.com/XiangRongLin) improved sharing a downloaded video to other apps, which crashed the app before this release. The app also crashed when you tried to download a really large video, which [@XiangRongLin](https://github.com/XiangRongLin) fixed in his third contribution. The seek duration setting was not properly shown in any other language then English, but [@B0pol](https://github.com/B0pol) let them magically appear again. And finally, [@kapodamy](https://github.com/kapodamy) took care of a nasty bugs, which made some WebM videos non seekable. Now you can seek all you want.
+
+All of the following fixes were done by the incredible [@mauriciocolli](https://github.com/mauriciocolli). Overall, he brought seven (!) different contributions in todays release. He solved an issue where the last screen of the app could be popping up during the app exit animation. A bug in the main screen resulted to tabs not being restored correctly if the app was put to sleep by Android. The main screen had another issue, especially with the menu, which could vanish until the user interacted with the app again. Sharing a YouTube link to NewPipe could end up crashing NewPipe if certain characters were in those links, not an issue anymore. And finally, he improved the handling of unavailable content on YouTube, which resulted in a crash previously.
+
+## Development
+
+This release also includes two big improvements of the development process. [@Stypox](https://github.com/Stypox) and [@mauriciocolli](https://github.com/mauriciocolli) changed the way `.apk` files are generated in the development process. This makes it way easier to test a change on your device, since you can have all of these NewPipe apps alongside each other, as long as you have the storage for them.
+
+[@Poolitzer](https://github.com/poolitzer), who already took over the latest blog posts, now created up-to-date issue templates, which makes it easier for you to report a bug or suggest a feature. The final form of them that you are experiencing is the result of a combined effort of the whole community, thanks for everyone who forced me to implement and discuss their suggestions.
+
+
+## Conclusion
+
+That’s it for this release. A big thanks goes out to all the people who made this possible, especially those not mentioned: Developers for helping out fellow developers, testers for testing, maintainers for maintaining, users like you for reporting the bugs in the first place. If you'd like to get a more technical view into the app, you can find the release page [here](https://github.com/TeamNewPipe/NewPipe/releases/tag/v0.19.0) and the corresponding PR [here](https://github.com/TeamNewPipe/NewPipe/pull/3267).
```

Comment:
Press F to pay respect for translators

Reply:
did we had big changes in this release for them? I only saw you messing around with plural strings when you fixed the second duration bug

Reply:
Look at commits from weblate: https://github.com/TeamNewPipe/NewPipe/pull/3267
And yeah, something big happened, **Nepali has been fully translated**, look at this huge commit https://github.com/TeamNewPipe/NewPipe/pull/3267/commits/373ec960513a9eac5a1b47f5141635e7fe836221

Anyway, I don't see any reason not to mention them when the sentence is about thanking the other contributors not mentionned in the post.

Reply:
Hm, I mention them then